sympy/solvers/diophantine.py

```diff
@@ -2062,58 +2062,61 @@ def _diop_ternary_quadratic_normal(var, coeff):
     return _remove_gcd(x_0, y_0, z_0)
 
 
-def square_factor(a):
+def sqf_normal(a, b, c, steps=False):
     """
-    Returns an integer `c` s.t. `a = c^2k, \ c,k \in Z`. Here `k` is square
-    free.
+    Return `a', b', c'`, the coefficients of the square-free normal
+    form of `ax^2 + by^2 + cz^2 = 0`, where `a', b', c'` are pairwise
+    prime and `\gcd(a', b', c')` is `1`.  If `steps` is True then
```

Comment:
if `a, b, c` are pairwise prime, then `gcd(a, b, c) ` is 1. Do we need to restate it?